src/app/content/highlights/components/HighlightsPopUp.spec.tsx

```diff
@@ -221,4 +225,35 @@ describe('MyHighlights button and PopUp', () => {
     expect(track).toHaveBeenCalled();
 
   });
+
+  it('changes colors based on book theme', () => {
+    const book = formatBookData(archiveBook, mockCmsBook);
+
+    store.dispatch(receiveUser(user));
+    store.dispatch(receiveBook(book));
+    store.dispatch(openMyHighlights());
+
+    const focus = jest.fn();
+    const addEventListener = jest.fn();
+    const querySelectorAll = jest.fn(() => []);
+    const createNodeMock = () => ({focus, addEventListener, querySelectorAll});
+
+    const component = renderer.create(<Provider store={store}>
+      <Services.Provider value={services}>
+        <MessageProvider>
+          <HighlightsPopUp />
+        </MessageProvider>
+      </Services.Provider>
+    </Provider>, {createNodeMock});
+
+    expect(component.toJSON()).toMatchSnapshot();
+
+    renderer.act(() => {
+      store.dispatch(closeMyHighlights());
+      store.dispatch(receiveBook({...book, theme: 'yellow'}));
+      store.dispatch(openMyHighlights());
+    });
+
+    expect(component.toJSON()).toMatchSnapshot();
```

Comment:
This theme change is not really visible in snapshots, right? Maybe just check for props on heading and close icon?

Reply:
Only the class name changes. I tried testing for background with `getComputedStyle` but it doesn't seem to contain information about it.